soc/espressif/esp32/default.ld

```diff
@@ -759,6 +759,7 @@ SECTIONS
   {
     _rodata_start = ABSOLUTE(.);
     _rodata_reserved_start = ABSOLUTE(.);
+    __rom_region_start = ABSOLUTE(.);
```

Comment:
shouldn't there be a `__rom_region_end` somewhere ?
Comment applies to other ld files as well.

Reply:
Thanks for pointing this out. TBH, those symbols are not used in the ESP32 port. I see them used on other Xtensa platforms, but it states that `__rom_region_start` and `__rom_region_end` delimit text AND data - note, that this is not possible to do in the case of Espressif Xtensa SoCs, because of those addresses are on separate address spaces therefor subtracting them doesn't give you an `__rom_region_size` as one would expect. I set it to delimit the `.flash.text` region in all SoCs.